chia/util/full_block_utils.py

```diff
@@ -322,3 +323,26 @@ def header_block_from_block(
         header_block += bytes(transactions_info)
 
     return header_block
+
+
+@dataclass(frozen=True)
+class PlotFilterInfo:
+    pos_ss_cc_challenge_hash: bytes32
+    cc_sp_hash: bytes32
+
+
+def plot_filter_info_from_block(buf: memoryview) -> PlotFilterInfo:
+    buf = skip_list(buf, skip_end_of_sub_slot_bundle)  # finished_sub_slots
+    buf = skip_uint128(buf)  # weight
+    buf = skip_uint32(buf)  # height
+    buf = skip_uint128(buf)  # total_iters
+    buf = skip_uint8(buf)  # signage_point_index
+    pos_ss_cc_challenge_hash = bytes32(buf[:32])
+    buf = skip_bytes32(buf)  # pos_ss_cc_challenge_hash
+    buf = skip_proof_of_space(buf)  # proof_of_space
+    # Optional[challenge_chain_sp_vdf]
+    if buf[0] == 0:
+        return PlotFilterInfo(pos_ss_cc_challenge_hash, pos_ss_cc_challenge_hash)
```

Comment:
this looks like a typo. I assume it's not, but I think it would help future readers of this code to have a comment explaining why the `pos_ss_cc_challange_hash` and `cc_sp_hash` are supposed to be the same in this case